.github/workflows/actionlint.yaml

```diff
@@ -0,0 +1,23 @@
+name: Actionlint
+
+on:
+  pull_request:
+  push:
+    branches: main
```

Comment:
should we restrict this to PRs that touch the workflows?

Reply:
That makes sense :+1:

Done that using a [path filter](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) in the additional commit `ci: run actionlint only when .github/workflows/** changes`.

@raboof> please make sure to check that the workflow indeed runs on this PR when you approve them.